modules/stanford_events_importer/stanford_events_importer.module

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/**
+ * @file
+ * File description.
+ *
+ * Long description.
+ */
+
+use Drupal\migrate\Plugin\MigrationInterface;
+use Drupal\migrate_tools\MigrateBatchExecutable;
+use Drupal\migrate\MigrateMessage;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Alter the config form to add the migrate_tools UI.
+ */
+function stanford_events_importer_form_config_pages_stanford_events_importer_form_alter(array &$form, FormStateInterface $form_state) {
```

Comment:
I don't like the idea of having the importer form added to the config pages form. maybe just add a link to the import page. less for us to manage in multiple places.

Reply:
I'd like to avoid having to expose the migrate UI pages to site owners as there are destructive tasks and it creates two places for the site owner to perform the import. How about changing this down into a much smaller set of actions and simply provide an import or update button instead of all the options?